vector/src/main/res/values-fr/strings_tchap.xml

```diff
@@ -54,6 +54,7 @@
     <string name="tchap_burger_menu_public_rooms">Salons forums</string>
     <string name="tchap_burger_menu_info">Cette application n’est pas approuvée\npour l’échange d’informations en diffusion restreinte</string>
     <string name="tchap_error_message_default">Une erreur est survenue, veuillez réessayer ultérieurement</string>
+    <string name="tchap_colon_append">&#160;:</string>
```

Comment:
Why the value is different in https://github.com/dinsic-pim/tchap-android-v2/pull/47/files#diff-78372f89158787476f179c9f282d56bed07996b94bfe3b752ccba96c6614ec4f

Reply:
because in French, a no-break space is expected before ":"